myhpom/validators.py

```diff
@@ -0,0 +1,36 @@
+import re
+from django.core.exceptions import ValidationError
+from django.core.validators import EmailValidator, RegexValidator
+
+# First Name, Last Name: At least one alphanumeric character.
+name_validator = RegexValidator(
+    regex=r'\w',
+    flags=re.U,
+    message='Please enter your name'
+)
+
+# Email: valid email address
+email_validator = EmailValidator()
+
+# Email is not already taken
+def email_not_taken_validator(email):
+    from myhpom.models import User
```

Comment:
This import is ultimately just `from django.contrib.auth.models import User` - using that directly would probably address whatever circular import required that this import get put here, and make it clearer which model User is.

Reply:
I could do that, but the reasons for importing myhpom.models.user.User everywhere are (1) we ensure that the pre_save is picked up, and (2) if we create our own subclass later, we don't have to edit every reference to the User model. 

The pre_save is using validators, which is why we end up with a circular commit. Maybe it's not necessary to do the pre_save validation, but I like the added safety. In which case, one of the imports has to be at runtime rather than import time.